messaging/WFLY-14133_configurable_bridge_call_timeout.adoc

```diff
@@ -0,0 +1,150 @@
+= Add call-timeout attribute to the Artemis core bridge resource
+:author:            Tomas Hofman
+:email:             thofman@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+The Artemis broker defines a call-timeout attribute on the core bridge, which sets a time out for blocking calls to the
+server. Unless modified, the call-timeout defaults to 30 seconds (this is the default value defined in Artemis).
+It is possible to programmatically modify this timeout by a lower level Artemis API, but it is not currently possible
+to modify with the high level API that Wildfly currently uses to create core bridges. It is currently not possible at all to
+modify this attribute by an EAP user / operator.
```

Comment:
What do you mean by EAP operator here? `EAP operator` is product that allows to start EAP on OpenShift. Does this rather means "EAP administrator"?

Reply:
Yes, I mean the person, not the product. Replaced by "administrator".